### DISCUSSION TITLE
js at head with attributes for json blocks

### DISCUSSION THREAD

**Discussion by gbowne-quickbase** [craftcms/cms#10219] [Ideas] [answered]
We have a few pages where we need to manually throw in some schema to account for ratings coming in from a third party, so we build out the schema.org object and produce a block like the one below, but we have to use `{% html at head %}` because `{% js at head %}` wraps the code in an additional `<script>` tag, and we can't use the with filter to add the `type="application/ld_+json"` attribute to it:
```
<script type='application/ld+json'>
{
    "@context": "http://schema.org",
    "@type": "SoftwareApplication",
    "name": "Company",
    "aggregateRating": {
        "@type": "AggregateRating",
        "ratingValue": "{{ ratings.ratingValue }}",
        "ratingCount": "{{ ratings.ratingCount }}",
    }
}
</script>
```

Ideally, we'd use `{% json at head with { type: "application/ld+json"} %}` to wrap around our object

**Comment by brandonkelly** [comment, marked as answer]
You can pull this off using the [`{% script %}`](https://craftcms.com/docs/3.x/dev/tags.html#script) tag:

```js
{% script at head with {type: 'application/ld+json'} %}
    "@context": "http://schema.org",
    "@type": "SoftwareApplication",
    "name": "Company",
    "aggregateRating": {
        "@type": "AggregateRating",
        "ratingValue": "{{ ratings.ratingValue }}",
        "ratingCount": "{{ ratings.ratingCount }}",
    }
{% endscript %}
```

**Reply by gbowne-quickbase** [reply]
Thanks. I hate when I don't read the documentation in enough detail!!
